en/docs/install-and-setup/setup/security/enabling-hostname-verification.md

```diff
@@ -36,3 +36,11 @@ The hostname verification is **disabled** by default. This is done using the `o
     -   **AllowAll:** This option turns off hostname verification for the server. Note that this is not recommended in a production setup and should only be used for demonstrations and testing.
 
 
+### Configuring hostname verification for WebSockets
+
+WebSocket hostname verification is **disabled** by default. In order to enable hostname verification for WebSockets the following configuration needs to be added to the `deployment.toml` file in `<PRODUCT_HOME>/repository/conf` .
+
+```toml
+[transport.wss.sender.parameters]
```

Comment:
@Lakith-Rambukkanage hope this is applicable to `[transport.ws.sender.parameters]` as well. Shall we mention that too, here?

Reply:
We have omitted `ws` since a certificate is not involved in this context.